helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/constraints/LeastPartitionCountConstraint.java

```diff
@@ -43,7 +43,7 @@ private LeastPartitionCountConstraint() {
    * clusterContext.
    */
   @Override
-  float getAssignmentOriginScore(AssignableNode node, AssignableReplica replica,
+  public float getAssignmentOriginScore(AssignableNode node, AssignableReplica replica,
```

Comment:
AssignmentOriginScore isn't too intuitive by itself, so could we add some comments for people to easily understand what it's about?